code_generation/namer.cpp

```diff
@@ -155,7 +155,7 @@ QString Namer::getClassName(const QString &elementName)
 {
     QString name;
     QStringList parts = removeInvalidCharacters(elementName).split("_");
-    foreach (QString part, parts) {
+    for (const QString &part : qAsConst(parts)) {
```

Comment:
This is correct, but it's simpler/better to just make `parts` const on line 157, like this: `const QStringList parts = ...`.
It also indicates (and even enforces) that `parts` isn't modified anywhere else in this method.

Reply:
> Sorry for being a bit late, national holiday in France :-)

No worries, I should apologise for merging this without review with wider audience. I will fix the mentioned issues an submit a PR. As I mentioned below: I think keeping the performance considerations could be valuable for this project.